### PR TITLE
Volunteers Routes

### PR DESCRIPTION
**Pull request by mboperator**
## Changes
- Removed nav links to Google Forms and Spreadsheets
- Created a template for Volunteer#List
- Filled in some volunteer boilerplate for CRUD

### CODE REVIEW

File: src/components/VolunteerListingCard.jsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            <Icon type="environment-o" style={{ marginRight: '5px' }}/>
+            {address}
+          </aside>
+        }
+      </HeaderContainer>
+
+      <Summary>
+        {/* <div>
+          {`${houseListing.beds} beds - ${prettyPrint('length_of_stay', houseListing.length_of_stay)}`}
+        </div> */}
```
Comment (michaelstephens) [inline src/components/VolunteerListingCard.jsx:149]: Should probably remove this

File: src/components/VolunteerListingCard.jsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+      <TagList>
+        {/* <Tag>{prettyPrint('paid', houseListing.paid)}</Tag>
+        <Tag>{prettyPrint('length_of_stay', houseListing.length_of_stay)}</Tag>
+        {houseListing.child_friendly &&
+          <Tag>Child Friendly</Tag>
+        }
+        {houseListing.pets_accepted &&
+          <Tag>Pets Accepted</Tag>
+        } */}
```
Comment (michaelstephens) [inline src/components/VolunteerListingCard.jsx:160]: Probably this too

File: src/screens/helping/HousingForm.jsx [hunk 3]
```diff
@@ -44,182 +45,362 @@ const StackInput = ({ required, children, label }) => (
...
+            />
+          </StackInput>
+
+          <StackInput required label="Beds Available:">
+            <Select
+              showSearch
+              style={{ width: '100%' }}
+              value={formData.bedsAvailable}
+              placeholder="Select number of beds available"
+              optionFilterProp="children"
```
Comment (michaelstephens) [inline src/screens/helping/HousingForm.jsx:88]: Children as in kids or children as in programming children? Lol

File: src/screens/looking_for_resources/VolunteerForm.jsx [hunk 4]
```diff
@@ -35,152 +60,286 @@ const StackInput = ({ required, children, label }) => (
...
+                filterOption={(input, option) => option.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0}
+              >
+                <Option value="goleta">Goleta</Option>
+                <Option value="ojai">Ojai</Option>
+                <Option value="santa_barbara">Santa Barbara</Option>
+                <Option value="camarillo">Camarillo</Option>
+                <Option value="ventura">Ventura</Option>
+                <Option value="thousand_oaks">Thousand Oaks</Option>
+              </Select>
+            </StackInput> */}
```
Comment (michaelstephens) [inline src/screens/looking_for_resources/VolunteerForm.jsx:150]: Me no like comments of code

File: src/screens/looking_for_resources/VolunteerForm.jsx [hunk 5]
```diff
@@ -35,152 +60,286 @@ const StackInput = ({ required, children, label }) => (
...
```
Comment (michaelstephens) [inline src/screens/looking_for_resources/VolunteerForm.jsx:274]: Same

**Review by michaelstephens — APPROVED** [review]
None are really blocking

File: src/screens/looking_for_resources/VolunteerForm.jsx (hunk 4 above)
Comment (jeyip) [reply src/screens/looking_for_resources/VolunteerForm.jsx:150]: LOL

**Pull request merged**